applications/ConstitutiveLawsApplication/constitutive_laws_application.cpp

```diff
@@ -348,9 +348,51 @@ void KratosConstitutiveLawsApplication::Register()
     KRATOS_REGISTER_CONSTITUTIVE_LAW("ThermalLinearPlaneStrain", mThermalLinearPlaneStrain);
     KRATOS_REGISTER_CONSTITUTIVE_LAW("ThermalLinearPlaneStress", mThermalLinearPlaneStress);
 
+   /// Damage
+    /* Small strain */
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("SmallStrainPseudoThermalIsotropicDamage3DVonMises", mSmallStrainPseudoThermalIsotropicDamage3DVonMises);
```

Comment:
Yupi, more laws!

Reply:
One thing that is bothering me is that I need to register some "idiotic CL" so that the combination of standard damage is templated so the "derived" one compiles...

Reply:
Yep :P